R/inbo_rapport.R

```diff
@@ -15,6 +15,10 @@
 #' @param fig_crop \code{TRUE} to automatically apply the \code{pdfcrop} utility
 #'   (if available) to pdf figures.
 #' @param pandoc_args Additional command line options to pass to pandoc
+#' @param babel What language(s) to use in the document.
+#' Only used with `style = "Flanders"`.
+#' If you specify multiple languages, the main language is the last one.
```

Comment:
> If you specify multiple languages, the main language is the last one.

Not so clear what this means exactly. Best add the actual implication of one's choice for a 'main language', that will probably clarify the meaning of this argument.

After all, the user chooses the language (s)he writes, hence it must have to do with some automated aspects (such as splitting words with hyphens I presume).

Reply:
This is how the babel package in LaTeX wants you to specify multiple languages. Only relevant is you need multiple languages in the document. (E.g. summary in a different language). It affects hyphenation and names of figures, tables, table of content, ...